code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -213,7 +213,7 @@
 	GLOB.cargo_ammo_vendors.Remove(src)
 
 /obj/machinery/vending/lasgun
-	name = "ColMarTech Lasgun Field Charger"
+	name = "ColMarTech lasrifle Field Charger"
```

Comment:
I'm not the biggest code diver but wouldn't it sound somewhat better if you just put the full name here instead of just "lasrifle"? It looks weird but is purely cosmetic.

Reply:
Removing `ColMarTech` would be positive as well.